docs/ref/contrib/auth.txt

```diff
@@ -212,7 +212,7 @@ Methods
 
         Returns ``False`` if
         :meth:`~django.contrib.auth.models.User.set_unusable_password()` has
-        been called for this user.
+        been called for this user or if password is empty.
```

Comment:
I'm not sure if "empty" is the best term. Does that mean an empty string or None (looking at the code, I see that it's None). The method also return False if the password uses an unsupported password hasher.